src/core/util/constant.js

```diff
@@ -7,9 +7,22 @@ angular.module('material.core')
  */
 function MdConstantFactory($sniffer) {
 
-  var webkit = /webkit/i.test($sniffer.vendorPrefix);
+  var isWebkit = /webkit/i.test($sniffer.vendorPrefix);
+  var prefixTestEl = document.createElement('div');
```

Comment:
This code is not working, because the `camelCase` function is converting snake case to camel case, but the properties are not in snake case, which means, that we need to modify this function.